frontend/tests/moderator/index.test.tsx

```diff
@@ -21,20 +21,36 @@ const tempArray = [
 
 function renderHome(props: Partial<IndexProps> = {}) {
   const defaultProps: IndexProps = {
-    queueData: [],
+    queueData: {
+      message: '',
+      articleData: [],
+    },
     duplicates: [],
   };
   return render(<Index {...defaultProps} {...props} />);
 }
 
-function renderHomeWithArticles(props: Partial<IndexProps> = {}) {
+function renderWithoutDuplicate(props: Partial<IndexProps> = {}) {
   const defaultProps: IndexProps = {
-    queueData: tempArray,
-    duplicates: [],
+    queueData: {
+      message: '',
+      articleData: tempArray,
+    },
+    duplicates: []
   };
   return render(<Index {...defaultProps} {...props} />);
 }
 
+function renderWithDuplicate(props: Partial<IndexProps> = {}) {
+    const defaultProps: IndexProps = {
+      queueData: {
+        message: '',
+        articleData: tempArray,
+      },
+      duplicates: ['dsfsdfsdfsdf']
```

Comment:
This branch is out of date, main has since changed `queueData: {message:'', articleData: Article[]}` -> just `queueData: Article[]`